### PR TITLE
first release notes

### PR DESCRIPTION
**Pull request by woutslakhorst**
Added sub-sections that should be present every release. Instead of filling them with issues, I added some explanation to them.

### CODE REVIEW

File: docs/pages/release_notes.rst [hunk 1]
```diff
@@ -5,6 +5,44 @@ Release notes
...
+***************
+Almond (v1.0.0)
+***************
+
+Release date: 2022-04-01
+
+This is the initial release of the Nuts node reference implementation.
+It implements RFC001 - RFC017 specified at the `Nuts specification <https://nuts-foundation.gitbook.io>`_.
+This release is intended for developers. It contains a stable API that will be backwards compatible for the next versions.
+The releases until the first production release will mainly focus on network and OPS related features.
```
Comment (stevenvegt) [inline docs/pages/release_notes.rst:17]: I think OPS should be written as Ops

File: docs/pages/release_notes.rst [hunk 2]
```diff
@@ -5,6 +5,46 @@ Release notes
...
-v1.0.0
-******
+***************
+Almond (v1.0.0)
+***************
+
+Release date: 2022-04-01
+
+This is the initial release of the Nuts node reference implementation.
+It implements RFC001 - RFC017 specified by the `Nuts specification <https://nuts-foundation.gitbook.io>`_.
```
Comment (gerardsn) [inline docs/pages/release_notes.rst:15]: RFC017 is network v2. This isn't part of nuts-node v1 right?

File: docs/pages/release_notes.rst [hunk 3]
```diff
@@ -5,6 +5,46 @@ Release notes
...
+This release is intended for developers. It contains a stable API that will be backwards compatible for the next versions.
+The releases until the first production release will mainly focus on network and OPS related features.
+
+To start using this release, please consult the getting started section.
```
Comment (gerardsn) [inline docs/pages/release_notes.rst:19]: Not sure if this is supposed to be in release notes, but perhaps a link to read the docs here?

File: docs/pages/release_notes.rst [hunk 4]
```diff
@@ -5,6 +5,46 @@ Release notes
...
+Deprecated features
+===================
+
+Some features will be deprecated because they have been succeeded by an improved version or when they are no longer used.
+Removing old code helps in reducing maintenance costs of the code base.
+Features that are marked as *deprecated* will be listed here.
+Any vendor using these features will have until next version to migrate to the alternative.
+Keep an eye on this section for every release.
+
+- VCR V1 API is deprecated. Please migrate all calls to the V2 API.
```
Comment (reinkrul) [inline docs/pages/release_notes.rst:44]:
```suggestion
- VCR V1 API is deprecated and will be removed in the next release. Please migrate all calls to the V2 API.
```

**Review by reinkrul — APPROVED** [review]
Small remark on deprecated section. Furthermore, not really much to say about it.

Maybe later we can add a link to the roadmap.

File: docs/pages/release_notes.rst (hunk 2 above)
Comment (woutslakhorst) [reply docs/pages/release_notes.rst:15]: true

File: docs/pages/release_notes.rst (hunk 3 above)
Comment (woutslakhorst) [reply docs/pages/release_notes.rst:19]: these notes are on read the docs...

**Pull request merged**